resources/assets/js/components/settings/ReminderRules.vue

```diff
@@ -0,0 +1,107 @@
+<style scoped>
+</style>
+
+<template>
+  <div class="reminder-rules">
+    <notifications group="main" position="bottom right" />
+
+    <h3 class="mb3">
+      {{ $t('settings.personalization_reminder_rule_title') }}
+    </h3>
+    <p>
+      {{ $t('settings.personalization_reminder_rule_desc') }}
+    </p>
+
+    <div class="dt dt--fixed w-100 collapse br--top br--bottom">
+
+      <div class="dt-row">
+        <div class="dtc">
+          <div class="pa2 b">
+            {{ $t('settings.personalization_contact_field_type_table_name') }}
+          </div>
+        </div>
+        <div class="dtc tr">
+          <div class="pa2 b">
+            {{ $t('settings.personalization_contact_field_type_table_actions') }}
+          </div>
+        </div>
+      </div>
+
+      <div class="dt-row bb b--light-gray" v-for="reminderRule in reminderRules" :key="reminderRule.id">
+        <div class="dtc">
+          <div class="pa2">
+            {{ $t('settings.personalization_reminder_rule_line', {count: reminderRule.number_of_days_before}) }}
+          </div>
+        </div>
+        <div class="dtc tr">
+          <div class="pa2">
+            <toggle-button :class="'reminder-rule-' + reminderRule.number_of_days_before" :value="reminderRule.active" :sync="true" :labels="true" v-on:change="toggle(reminderRule)" />
+          </div>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</template>
+
+<script>
+    import { SweetModal, SweetModalTab } from 'sweet-modal-vue';
+
+    export default {
+        /*
+         * The component's data.
+         */
+        data() {
+            return {
+                reminderRules: [],
+            };
+        },
+
+        components: {
+            SweetModal,
+            SweetModalTab
+        },
+
+        /**
+         * Prepare the component (Vue 1.x).
+         */
+        ready() {
+            this.prepareComponent();
+        },
+
+        /**
+         * Prepare the component (Vue 2.x).
+         */
+        mounted() {
+            this.prepareComponent();
+        },
+
+        methods: {
+            /**
+             * Prepare the component.
+             */
+            prepareComponent() {
+                this.getReminderRules();
+            },
+
+            getReminderRules() {
+                axios.get('/settings/personalization/reminderrules/')
```

Comment:
Carefull: there is a trailing slash at the end of the request. I think it will not work on apache !

Reply:
Good catch!